src/main/org/codehaus/groovy/tools/DgmConverter.java

```diff
@@ -37,7 +37,7 @@
 
 public class DgmConverter implements Opcodes {
 
-    public static void main(String[] args) throws IOException, ClassNotFoundException {
+    public static void main(String[] args) throws IOException {
```

Comment:
even though this is not likely to cause a problem, still removal of checked exception on public method